docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,929 @@ The LDML specification is divided into the following parts:
+
+```
+person_name ⇒ 
+   prefix 	⇒ "г-н" 		// "Mr"
+   given 	⇒ "Иван"		// "Ivan"
+   given2 	⇒ "Петрович"	// "Petrovich"
+   surname ⇒ "Васильев"	// "Vasiliev"
+```
+
+In Slavic languages, when _`addressing`_ a person (with `length="long"`), it might be
+
+* г-н Иван Петрович Васильев `// "Mr Ivan Petrovich Vasiliev"`
+
+And when _`referring`_ to a person, it might place the surname first.:
+
+* Васильев Иван Петрович `// "Vasiliev Ivan Petrovich"`
+
+The `monogram` usage is for very short abbreviated names, such as might be found in online messaging text avatars or other annotations. Ideally, a `monogram` format should result in something that could fit in an em square. Some emoji provide examples of this: 🅰️ 🆎 🆘 
+
+When used with `length`, for many alphabetic locales a `monogram` would resolve to one, two, or three characters for short, medium, and long respectively. But that may vary depending on the usage in a locale.
+
+### 4.4 <a name="4-4-formality" href="#4-4-formality">formality</a>
+
+The `formality` indicates the formality of usage. A name on a badge for an informal gathering may be much different from an award announcement at the Nobel Prize Ceremonies.
+
+Note that the formats may be the same for different formality scenarios depending on the length, usage, and cultural conventions for the locale. For example short formal and short informal may both be just the given name.
+
+| Parameter  | Description |
+| ---------- | ----------- |
+| `formal`   | A more formal name for the individual. The composition depends upon the language. For example, a particular locale might include the prefix and suffix and a full middle name (given2) in the long form.<br><br>`length="medium", formality="formal"`<br>“Robert J. Smith” |
+| `informal` | A less formal name for the individual. The composition depends upon the language. For example, a language might exclude the prefix, suffix and given2 (middle) name. Depending on the length, it may also exclude the surname. The formatting algorithm should choose any passed in name data that has an _informal_ attribute, if available.<br><br>`length="medium", formality="informal"`<br>“Bob Smith” |
+
+## 5 <a name="5-namepattern-syntax" href="#5-namepattern-syntax">namePattern Syntax</a>
+
+A _namePattern_  is composed of a sequence of field IDs, each enclosed in curly braces, and separated by zero or more literal characters (eg, space or comma + space). An Extended Backus Normal Form (EBNF) is used to describe the namePattern format for a specific set of attributes. It has the following structure. This is the `( #PCDATA )` reference in the element specification above.
+
+|              | EBNF           | Comments    |
+| ------------ | -------------- | ----------- |
+| namePattern  | = literal?<br>( modField  literal? )+; | Two literals cannot be adjacent |
+| modField     | = '{' field modifierList? '}';         | Parts of a name, described below |
```

Comment:
Change "Parts of a name, described below" to a name field, optionally modified.

Reply:
Done